modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -116,7 +116,7 @@ namespace {{packageName}}.Client
 
         // Creates and sets up a RestRequest prior to a call.
         private RestRequest PrepareRequest(
-            String path, {{^netStandard}}RestSharp.{{/netStandard}}Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
@wing328 I'm pretty sure the `{{^netStandard}}` is required because of the differences between RestSharp and RestSharp.Portable.  iirc, we were holding off on #2790 until version 2.3.0 because of the breaking change in the method signature.

Do you think it would make sense to provide an overloaded signature with the older Dictionary type to avoid breaking consumers that rely on this method?